src/eml/main.ml

```diff
@@ -21,10 +21,16 @@ struct
   let workspace_path =
     ref ""
 
+  let reason_syntax =
+    ref false
+
   let options = Arg.align [
     "--workspace",
     Arg.Set_string workspace_path,
     "PATH Relative path to the Dune workspace for better locations";
+    "--syntax-reason",
```

Comment:
I recommend `--emit-reason` for the name of this parameter (based on the help string), because `syntax-reason` makes me, as the reader, think of something that is required from me (which syntax to use), but this is rather about what syntax the templter will emit.